src/sandstorm/web-session.capnp

```diff
@@ -208,6 +291,11 @@ interface WebSession @0xa50711a14d35a8ce extends(Grain.UiSession) {
         language @3 :Text;  # Content-Language header (optional).
         mimeType @4 :Text;  # Content-Type header.
 
+        dav @18 :Text;
```

Comment:
Is the DAV header only returned for OPTIONS queries? That is what the standard implies. If so, then if we create an `isDav()` method like I suggested above rather than passing through OPTIONS, then we don't need this field here anymore.

If it is returned for other request types, then I wonder if it is really only returned in success cases and not error cases. I wonder if it would make sense to promote to the top level of `Response`.